pkg/Library/Dom/Common/common.js

```diff
@@ -12,7 +12,7 @@ import './draggable-item.js';
 import './drop-target.js';
 import './container-layout.js';
 //
-//import '../Material/material-xen/material-xen.js';
+import '../Material/material-xen/material-xen.js';
```

Comment:
Can you move `image-upload` to Library/Dom? ... material-xen has old stuff in it.

Reply:
I moved it into `Library/Dom/Common` instead (because there is nothing else in `Library/Dom`, so I thought this is what you meant).